layouts/shortcodes/doc-versions-list.html

```diff
@@ -0,0 +1,14 @@
+{{ $versions := .Page.Param "versions" }}
+{{ $thisPageRelUri := .Page.RelPermalink }}
+{{ $thisPageTitle := .Page.Title }}
+{{ $thisVersionArray := split (.Page.Param "version") "." }}
+
+<ul>
+{{ range $index, $version := $versions }}
+    {{ if ne .version ( delimit $thisVersionArray "." ) }}
+        <li>
+            <a href="{{ .url }}{{ $thisPageRelUri }}">{{ $thisPageTitle }} {{ T "for_kubernetes" }} {{ .version }}</a>
```

Comment:
Not sure how well this will localize - we might want to reimplement it with, eg a format string that each localization owns.

Seems good for now.